csrc/codegen.cpp

```diff
@@ -493,6 +493,9 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
           NVF_ERROR(dtype == DataType::ComplexDouble);
           code_ << "std::complex<double>" << value;
         }
+      } else if (std::holds_alternative<ArrayType>(s->dtype().type)) {
+        // print out the vector of numbers.
+        code_ << to_str(s->dtype()) << stringifyPolymorphicValAsVector(value);
```

Comment:
Should we put the entire thing, starting from line 469 into this function?

Reply:
I had missed this comment. This makes sense. Let me update the PR.